src/execution_plan.c

```diff
@@ -1812,7 +1822,7 @@ int ExecutionPlan_ExecutionGet(RedisModuleCtx *ctx, RedisModuleString **argv, in
             ".collect()"
             ".flatmap(lambda x: [i for i in x])"
             ".run(convertToStr=False, collect=False)", eid);
-        int res = RedisGearsPy_ExecuteReturnResultsOnly(ctx, fargv, 2);
```

Comment:
you allocate the fargv with RedisModule_Calloc should be RG_ALLOC

Reply:
Done in f8911ec (also fixed to use RG_FREE instead).